packages/labs/examples/timezoneSelectExtendedExample.tsx

```diff
@@ -0,0 +1,232 @@
+    private handleUseDefaultChange = handleBooleanChange((useDefault) =>
+        this.setState({ useDefault }));
+    private handleDefaultToUserTimezoneGuessChange = handleBooleanChange((defaultToUserTimezoneGuess) =>
+        this.setState({ defaultToUserTimezoneGuess }));
+    private handleUseCustomRendererChange = handleBooleanChange((useCustomRenderer) =>
+        this.setState({ useCustomRenderer }));
+    private handleFormatChange = handleStringChange((targetDisplayFormat: TimezoneDisplayFormat) =>
+        this.setState({ targetDisplayFormat }));
+
+    protected renderExample() {
+        const {
+            date,
+            time,
+            timezone,
+            dateTime,
+            targetDisplayFormat,
+            disabled,
+            showLocalTimezone,
+            useCustomRenderer,
+            useDefault,
+            defaultToUserTimezoneGuess,
+        } = this.state;
+
+        const selection = dateTime && timezone
+            ? toISOString(dateTime, timezone)
+            : "Select a date, time, and timezone";
+
+        return (
+            <div>
+                <div
+                    className={Classes.ELEVATION_1}
+                    style={{ display: "inline-block", background: "white", borderRadius: 3 }}
+                >
+                    <DatePicker
+                        value={date}
+                        onChange={this.handleDateChange}
+                    />
+
+                    <div style={{ padding: 10, display: "flex", justifyContent: "center" }}>
+                        <TimePicker
+                            value={time}
+                            onChange={this.handleTimeChange}
+                            precision={TimePickerPrecision.SECOND}
+                        />
+
+                        <TimezoneSelect
+                            date={date}
+                            value={timezone}
+                            onChange={this.handleTimezoneChange}
+                            targetDisplayFormat={targetDisplayFormat}
+                            showLocalTimezone={showLocalTimezone}
+                            disabled={disabled}
+                            defaultValue={useDefault ? EXAMPLE_DEFAULT_TIMEZONE : undefined}
+                            defaultToLocalTimezone={defaultToUserTimezoneGuess}
+                            targetRenderer={useCustomRenderer ? this.targetRenderer : undefined}
+                        />
+                    </div>
+                </div>
+
+                <div style={{ marginTop: 20 }}>
+                    <Tag
+                        className={classNames(Classes.MINIMAL, Classes.LARGE)}
+                        onRemove={this.reset}
+                    >
+                        <Icon iconName="time" />
+                        <span style={{ marginLeft: 10 }}>{selection}</span>
+                    </Tag>
+                </div>
+            </div>
+        );
+    }
+
+    protected renderOptions() {
```

Comment:
please move all simple prop switches to the basic example, to demonstrate that they're built-in to the component (not a result of "extended" behavior).